core/src/main/java/org/infinispan/manager/DefaultCacheManager.java

```diff
@@ -809,7 +810,7 @@ private void internalStop() {
          }
 
          // We can stop the manager
-         log.infof("Stopping cache manager %s on %s", configurationManager.getGlobalConfiguration().transport().nodeName(), getAddress());
+         log.infof("Stopping cache manager %s on %s", globalConfiguration.transport().nodeName(), getAddress());
```

Comment:
```suggestion
         log.infof("Stopping cache manager %s on %s", globalConfiguration.cacheManagerName(), getAddress());
```
it would be nice to change it to `debug` like other start/stop cache manager log entries